packages/core/src/stepFunctions/workflowStudio/activation.ts

```diff
@@ -12,4 +12,20 @@ import { WorkflowStudioEditorProvider } from './workflowStudioEditorProvider'
  */
 export async function activate(extensionContext: vscode.ExtensionContext): Promise<void> {
     extensionContext.subscriptions.push(WorkflowStudioEditorProvider.register(extensionContext))
+
+    // Open the file with Workflow Studio editor in a new tab, or focus on the tab with WFS if it is already open
+    extensionContext.subscriptions.push(
+        vscode.commands.registerCommand('aws.stepfunctions.openWithWorkflowStudio', async (uri: vscode.Uri) => {
```

Comment:
Use our wrappers instead, `Commands.register` and `Commands.declare`.
```suggestion
        Commands.register('aws.stepfunctions.openWithWorkflowStudio', async (uri: vscode.Uri) => {
```